src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -110,7 +110,11 @@ public void removePerson(Person key) {
 
     //// util methods
 
+    /**
+     * Sorts the address book's internal list according to the given comparator
+     */
     public void sortPerson(Comparator<Person> c) {
+        assert persons.asUnmodifiableObservableList().size() > 0;
```

Comment:
Will this fail if a sort is performed on an empty Athena? Perhaps, it should instead just not do anything if the list size is 0.